xstream/src/test/com/thoughtworks/xstream/io/naming/NoNameCoderTest.java

```diff
@@ -0,0 +1,27 @@
+package com.thoughtworks.xstream.io.naming;
+
+import com.thoughtworks.xstream.io.naming.NoNameCoder;
+import junit.framework.TestCase;
+
+public class NoNameCoderTest extends TestCase {
+
+  public void testDecodeAttribute() {
+    final NoNameCoder noNameCoder = new NoNameCoder();
+    assertEquals("   ", noNameCoder.decodeAttribute("   "));
```

Comment:
Are spaces good test data? (generally for the whole PR the test data is not great)

Reply:
Fixed to something slightly more readable.